static/style.css

```diff
@@ -0,0 +1,183 @@
+}
+/*Side container for search and login in index.html*/
+.left-container {
+  background-color: #ffe6d3f1;
+  height: 100%;
+  width: 400px;
+  text-align: center;
+  border-radius: 10px;
+  margin-right: 20px;
+}
+/*Right side container for search results formatting*/
+.right-container {
+  background-color: #ffe6d3f1;
+  font-weight: bold;
+  height: 100%;
+  width: 60%;
+  margin: 0 auto;
+  text-align: center;
+  border-radius: 10px;
+}
+/*Ingredients container*/
+.fav-ingredients {
+  color: rgb(39, 39, 39);
+  background-color: white;
+  width: 95%;
+  margin: 0 auto;
+  border-radius: 4px;
+  text-align: left;
+}
+/*Recipes container*/
+.fav-recipes {
+  color: rgb(39, 39, 39);
+  background-color: white;
+  width: 95%;
+  margin: 0 auto;
+  border-radius: 4px;
+  text-align: left;
+}
+/*Individual recipe/ingredient container*/
+.recipe {
+  background-color: white;
+  color: rgb(75, 74, 74);
+  margin: 5px;
+  padding: 5px;
+  border-style: solid;
+  border-color: rgb(194, 191, 191);
+  border-width: 1px;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: row wrap;
+  text-align: left;
+}
+/*Inner description div for recipe/ingredient container*/
+.description {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+/*logo formatting*/
+.logo {
+  margin-top: 20px;
+  margin-bottom:50px;
+}
+/*All forms styling*/
+form {
+  margin-bottom: 70px;
+}
+/*Searchbox styling*/
+input[type=text] {
+  padding: 8px 10px;
+  margin: 5px 100px 5px 100px;
+  /* margin-left: 100px;
+  margin-right: 100px; */
+  font-size: 14px;
+  border: none;
+  border-radius: 4px;
+  text-align: center;
+}
+/*Searchbox styling*/
+input[type=password] {
+  padding: 8px 10px;
+  margin-left: 100px;
+  margin-right: 100px;
+  font-size: 14px;
+  border: none;
+  border-radius: 4px;
+  text-align: center;
+}
+/*Div containing filter options*/
+.filters {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+}
+/*Filter button styling*/
+.filter-button {
+  margin: 10px;
+  padding: 4px 6px;
+  background: #FFB659;
+  font-size: 14px;
+  border: none;
+  cursor: pointer;
+  border-radius: 4px;
+}
+.filter-button:hover {
+  background: #c46d04;
+}
+/*Search button styling*/
+button {
+  padding: 6px 10px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  margin-left: 100px;
+  margin-right: 100px;
+  background: #FB8C00;
+  font-size: 17px;
+  border: none;
+  cursor: pointer;
+  border-radius: 20px;
+}
+button:hover {
+  background: #c46d04;
+}
+/*account container styling*/
+.account {
+  margin-top: 150px;
+}
+/*padding between login and signup*/
+.account a {
+  margin: 30px;
+}
+/*Click styling for hyperlink*/
+.account a:link {
+  color: rgb(88, 86, 86);
+}
+.account a:visited {
+  color: black;
+}
+/*Google login button styling*/
+.google-img {
+  background-image: url("../static/btn_google_signin_light_normal_web.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-top: 30px;
+  height:50px;
+  width: 204px;
```

Comment:
Not sure if changing these values to be relative/percentage values would mess anything up, but if not then I would make these relative too.